packages/api/apollo-server/mutations/user.ts

```diff
@@ -0,0 +1,25 @@
+import { users } from '../mock-data'
+
+export interface IEditUser {
+  id: String
+  firstName?: String
+  lastName?: String
+  utorID?: String
+  email?: String
+  userType?: String
+  lastLogin?: String
+  isActive?: Boolean
```

Comment:
deactivate will be a separate mutation

Reply:
> deactivate will be a separate mutation

We have `isActive` for User and Assignment, and we have `isDeleted` for Annotation. Shall we use the either one for all the types?